src/Liip/RMT/Context.php

```diff
@@ -104,8 +106,12 @@ public function getList($id)
     protected function instanciateObject($objectDefinition)
     {
         list($className, $options) = $objectDefinition;
+        $o = new $className($options);
+        if ($o instanceof InputAwareInterface) {
+            $o->setInput($this->getService('input'));
```

Comment:
this seems to be the only place where we instantiate commands. lets use the magic interface here.